integr8ly/integreatly-operator
# MGDAPI-5245 Fix and unskip the E05 and E10 tests

# Issue link


https://issues.redhat.com/browse/MGDAPI-5245

# What


E05 fix - the secret is now found just based on the fact that it lives under Observability ns and has a correct name prefix. The previous way did not work because a different secret (test-dockercfg-<hash>) was only present as an SA secret for some reason (and this secret references the desired one (test-token-<hash> containing the token)). This test fails in Prow so it is skipped if running in Prow.

E10 unskip - it passed for me locally several times in a row so let's see how stable it is via nightly pipelines

# Verification steps




Eye review
Passing e2e test should suffice
(Optional) If you have a cluster you can execute:
LOCAL=false INSTALLATION_TYPE=managed-api TEST="E05" make test/e2e/single
LOCAL=false INSTALLATION_TYPE=managed-api TEST="E09" make test/e2e/single

I did that and both passed OK
